src/index.ts

```diff
@@ -106,7 +106,23 @@ export default async function (manager: Manager, settings: ComponentSettings) {
     handler('signup', event, settings)
   })
 
-  manager.addEventListener('watchVideo', event => {
+  manager.addEventListener('viewcategory', event => {
+    handler('viewcategory', event, settings)
+  })
+  manager.addEventListener('watchvideo', event => {
```

Comment:
Should those be all lowercase?

Reply:
@tomkln are you referring to the event names "viewcategory", "watchvideo", etc? If so then yes.

Reply:
Awesome, looks good to me then :)